WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticListenerSubscriber.cs

```diff
@@ -2,19 +2,29 @@
 {
     using System;
     using System.Diagnostics;
+    using System.Diagnostics.Tracing;
     using Microsoft.ApplicationInsights.Common;
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
 
-    internal sealed class AzureSdkDiagnosticListenerSubscriber : DiagnosticSourceListenerBase<object>
+    internal sealed class AzureSdkDiagnosticListenerSubscriber : DiagnosticSourceListenerBase<object>, IDisposable
     {
         public const string DiagnosticListenerName = "Azure.";
+        private readonly IDisposable logsListener;
 
         public AzureSdkDiagnosticListenerSubscriber(TelemetryConfiguration configuration) : base(configuration)
         {
+            // listen to Cosmos EventSource only - other logs can be sent using ILogger
+            this.logsListener = new AzureSdkEventListener(this.Client, EventLevel.Warning, "Azure.Cosmos");
```

Comment:
Can we make Event level user configurable?

Reply:
yes, we can, it might still be a bit difficult for users to configure it. It seems we also have a configuration on the Cosmos SDK side and it'd be nice to have all control on one SDK side and avoid inconsistent config, what do you think?

Reply:
At SDK side customer cannot control which type of event needs to be listened. 
SDK will emit events on the basis of listener's configuration. So, customer has to configure at AI SDK which kind of events they want to listen, and SDK will emit accordingly.
From SDK customer can only control the feature enablement.

Reply:
> At SDK side customer cannot control which type of event needs to be listened.
But they control which events are emitted with enablement flag AND thresholds, no? 

Can you please explain the scenario that you want to enable?
customer can't configure Cosmos SDK, but can  configure AI SDK?

AI SDK enablement tends to be one line of code, most customers will struggle to configure things beyond that, especially when they use codeless attach, I'd rather avoid adding extra configuration if there is no good scenario or need.

Reply:
if I am not mistaken, here you are capturing only Event Level as Warning? What will happen if SDK emits events in Event level as Information? will those get captured?

Reply:
another question about this hardcoded "Azure.Cosmos" I am wondering after this PR we won't be able to change this name? I think we should be fine with this as we won't be able to change this name in CosmosDb SDK also.

Reply:
Discussed with @sourabh1007 offline:
- AI SDK should listen to Info events by default. No config necessary
- Cosmos SDK owns tracing configuration (enablement, threshold to record diagnostics, etc), so it should also allow customers to enable info events.

This way all cosmos tracing configurations will stay in one place and would apply to AI SDK, OTel and custom listeners.